producers.md

```diff
@@ -317,20 +317,20 @@ While a graph is running, its current state will be able to be dumped.
 
 <!-- References -->
 
-[Component]: http://google.github.io/dagger/api/latest/dagger/Component.html
-[Dagger 2]: http://google.github.io/dagger/
+[Component]: api/latest/dagger/Component.html
+[Dagger 2]: ./
```

Comment:
The alternatives would be to inline the link, or link to `index` but that'd be the only reference to that page as `index` instead of `./`. See also in `_layout/default.html`.